src/current/v25.1/understand-hotspots.md

```diff
@@ -0,0 +1,313 @@
+
+#### Resolving Index Hotspots
+
+The resolution of the index hotspot often depends on your requirements for the data. If the sequential nature of the index serves no purpose, it is recommended to change the writes into the index to be randomly distributed. Ideally, primary keys in this instance would be set to `UUID`s, if your tolerance for swapover or even downtime allows it.
+
+If inserting in sequential order is important, the index itself can be [hash-sharded]({% link {{ page.version.version }}/hash-sharded-indexes.md %}), which means that it is still stored in order, albeit in some number of shards. Consider a `users` table, with a primary key `id INT`, which is hash-sharded with 4 shards, and a hashing function of modulo 4. The following image illustrates this example:
+
+<img src="{{ 'images/v25.1/hotspots-figure-4.png' | relative_url }}" alt="Hash-sharded index example" style="border:1px solid #eee;max-width:100%" />
+
+Now the writes are distributed into the tails of the shards, rather than the tail of the whole index. This benefits write performance but makes reads more challenging. If you need to read a subset of the data, you will have to scan each shard of the index.
+
+#### Paradoxical Performance Implications
+
+A note on the index hotspot is that they paradoxically increase the efficiency of CockroachDB from the standpoint of the number of CPU cycles per write. This is primarily due to [write compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction). As index hotspots continually move in a single direction, the [SST (sorted string table)]({% link {{ page.version.version }}/architecture/storage-layer.md %}#log-structured-merge-trees) files generated by them have little overlap. Consequently, this results in minimal write throughput compared to a properly distributed workload. However, this performance boost has a limit, and relying on hotspot behavior for this advantage would be short-sighted due to the other problems hotspots can cause.
+
+#### Lookback Hotspots
+
+**Synonyms:** index scan hotspot, moving read hotspot
+
+A _lookback hotspot_ is a type of index hotspot in which a moving subset of an index is being read in a way that burdens the cluster.
+
+For example, consider querying a `posts` table for the most recently created post: 
+
+~~~ sql
+CREATE TABLE posts(
+   id UUID,
+   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+   INDEX (created_at) USING HASH
+);
+
+SELECT MAX(created_at) FROM posts GROUP BY created_at ORDER BY created_at LIMIT 1;
+~~~
+
+What makes lookback hotspots unique is that the cluster is not hot by write, but hot by read. Separately lookback hotspots also tend not to specify a key, which would evade systems using key requests to identify hotspots.
+
+#### Queuing Hotspot
```

Comment:
oh nice, this is the queue-like workload section

suggest linking the `gc.ttlseconds` docs I mentioned earlier to this section